src/tx/claimReward.ts

```diff
@@ -271,6 +271,9 @@ export async function applyClaimRewardTx(
   mustUseAdminCert = false
 ): Promise<void> {
   if (ShardeumFlags.VerboseLogs) console.log(`Running applyClaimRewardTx`, tx, wrappedStates)
+
+  if (!AccountsStorage.cachedNetworkAccount?.current.utilityFlags.enableRewardTXs) return
```

Comment:
wanted to call something out for andrew. i am not using a strict equality check here so that is for some reason, cachedNetworkAccount is undefined, we return. else we would keep going